src/main/kotlin/com/ampnet/identityservice/service/impl/VerificationServiceImpl.kt

```diff
@@ -51,6 +53,10 @@ class VerificationServiceImpl : VerificationService {
         }
     }
 
+    private fun generateEip191Message(message: ByteArray): ByteArray =
+        0x19.toByte().toByteArray() + 0x45.toByte().toByteArray() +
+            ("thereum Signed Message:\n" + message.size).toByteArray() + message
```

Comment:
"Ethereum" not "thereum"

Reply:
This is copied from the library:
https://github.com/komputing/KEthereum/blob/master/eip191/src/main/kotlin/org/kethereum/eip191/EIP191.kt